source/_posts/CSS-Style-Guide.md

```diff
@@ -0,0 +1,277 @@
+---
```

Comment:
I'm assuming this is a _copy/paste_ from https://github.com/airbnb/css, is it? If so, I'd rather only link to the original source instead of bringing it in as a _post_ in order to prevent our "copy" from becoming _stale_ and also avoid duplicating 3rd party content. Alternatively, if we feel we need to have our own rules on top of Airbnb's guide, I'd suggest to _fork_ the upstream repo and _maintain_ our own fork as a repo.

Thoughts?

Reply:
👍